src/ztworldmgr.rs

```diff
@@ -228,6 +228,11 @@ pub fn get_entity_type_by_id(id: u32) -> u32 {
 
     i -= 1;
 
+    // define the types we are interested in (full impl soon)
```

Comment:
Change this to start with `// TODO: ` if you intend on doing it in a future PR, make sit easier to track so we don't release any modules with TODO still in them

Reply:
Ah yes, good call.